src/store/reducers/cohorts.reducer.js

```diff
@@ -1,14 +1,44 @@
-import { COHORTS } from '../actions/actions.type';
+import { COHORTS, SINGLE_COHORT } from '../actions/actions.type';
 
 export const INITIAL_STATE = {
-  cohortApplicants: {}
+  cohortApplicants: {},
+  cohortData: {
+    isSingleCohortPending: false,
+    isSingleCohortFailure: false
+  }
 };
 
-export const cohortsReducer = (state = INITIAL_STATE, action = {}) => {
+export const cohortsReducer = (state = {}, action = {}) => {
   switch (action.type) {
     case COHORTS.SET_DATA:
       return { ...state, cohortApplicants: action.payload };
     default:
       return state;
   }
 };
+
+export const cohortDataReducer = (state = {}, action = {}) => {
```

Comment:
Please separate out to a different file with its own INITIAL_STATE.

Btw, why is INITIAL_STATE being defined above but not used in either of the reducer functions anymore?